homeassistant/components/switch/unifi.py

```diff
@@ -0,0 +1,222 @@
+"""
+Support for devices connected to UniFi POE.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/switch.unifi/
+"""
+
+import asyncio
+import logging
+
+from datetime import timedelta
+
+from homeassistant.components import unifi
+from homeassistant.components.switch import SwitchDevice
+from homeassistant.components.unifi.const import (
+    CONF_CONTROLLER, CONF_SITE_ID, CONTROLLER_ID)
+from homeassistant.const import CONF_HOST
+from homeassistant.core import callback
+from homeassistant.helpers.device_registry import CONNECTION_NETWORK_MAC
+
+DEPENDENCIES = ['unifi']
+SCAN_INTERVAL = timedelta(seconds=15)
+
+LOGGER = logging.getLogger(__name__)
+
+STORAGE_KEY = 'unifi.poe_off_clients'
+STORAGE_VERSION = 1
+SAVE_DELAY = 10
+
+
+async def async_setup_platform(
+        hass, config, async_add_entities, discovery_info=None):
+    """Component doesn't support configuration through configuration.yaml."""
+    pass
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up switches for UniFi component.
+
+    Switches are controlling switch ports with Poe.
```

Comment:
You could call it "network switch" for better distinction from "switch" as in home assistant switches

Reply:
But it is a switch?

Reply:
Now I see what you mean...